internal/app/http/service.go

```diff
@@ -213,7 +213,7 @@ func (dhs *defaultHandlerService) DeleteThingModel(ctx context.Context, repo str
 }
 
 func (dhs *defaultHandlerService) GetCompletions(ctx context.Context, kind string, args []string, toComplete string) ([]string, error) {
-	rs, err := repos.GetSpecdOrAll(dhs.serveRepo)
+	rs, err := repos.GetUnion(dhs.serveRepo)
```

Comment:
At all places where we call repos.GetUnion(), we should name the variable `u` or `union`.
(as it is already done in completion/complete.go).
It makes the code better readable.

Reply:
done